workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py

```diff
@@ -164,7 +165,9 @@ def plot_cubed_sphere_map(
     )
 
 
-def plot_histogram(run_diags: RunDiagnostics, varname: str) -> RawHTML:
+def plot_histogram(
+    run_diags: RunDiagnostics, varname: str, xscale="linear", yscale="linear"
```

Comment:
Did you want to keep the original default of "log"?

Reply:
I figured a default more in keeping with matplotlib defaults was worth the breaking change. I modified the usage for the precip histogram in static_report.py